py3status/modules/mpris.py

```diff
@@ -375,10 +376,23 @@ def player_on_change(*args):
             """
             data = args[1]
             status = data.get('PlaybackStatus')
-            if status:
-                player = self._mpris_players[player_id]
-                player['status'] = status
-                player['_state_priority'] = WORKING_STATES.index(status)
+            if not status:
+                return
+
+            player = self._mpris_players[player_id]
+
+            # Note: Workaround. Since all players get noted if playback status
+            #       has been changed we have to check if we are the choosen one
+            try:
+                dbus_status = player['_dbus_player'].PlaybackStatus
+            except GError:
+                # Prevent errors when calling methods of deleted dbus objects
+                return
+            if status != dbus_status:
```

Comment:
With this if statement we stop an title change. But how can we find out if we are the right player here?